apps/web/src/pages/api/v1/profile/index.ts

```diff
@@ -42,6 +44,26 @@ handler.post(
       throw new BadRequest('Could not create account')
     }
 
+    // Check if the account is the admin account, and if so set admin permissions
+    if (body.email === Environment.firebaseAdminEmail) {
```

Comment:
Does this mean only a single user can be admin?

Reply:
I assumed we would be supporting adding other admin users in the near future, and there is an endpoint for updating the claims ready to be used, but I wasn't sure how we should display that to the admin user. I'm going to connect with Tom on that.